signals/links/google_open_redirect.yml

```
@@ -0,0 +1,6 @@
+name: "Link: Google Open Redirect"
+type: "query"
+source: |
+  regex.match(subject.subject, "^[^a-z]*$")
```

```suggestion
  regex.match(subject.subject, "^[^a-z ]*$")
```
adding space to char set, same as upper letters?